src/common/dcsvg.cpp

```diff
@@ -46,35 +46,40 @@ namespace
 // C locale (i.e. always using "." for the decimal separator) and with the
 // fixed precision (which is 2 for some unknown reason but this is what it was
 // in this code originally).
-inline wxString NumStr(double f)
+inline wxString NumStr(double const& f)
```

Comment:
Sorry, why should we pass `double` by reference? This is very unusual and, AFAIK, has no advantages compared to passing by value (which will use an FP register normally).